tests/replicaset/readconcern-001.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+ReadConcern: MongoDB\Driver\Manager::executeQuery() with readConcern option
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(REPLICASET) ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$bulk = new MongoDB\Driver\BulkWrite();
+$bulk->insert(['_id' => 1, 'x' => 1]);
+$bulk->insert(['_id' => 2, 'x' => 2]);
+$manager->executeBulkWrite(NS, $bulk);
+
+$rc = new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LOCAL);
+$query = new MongoDB\Driver\Query(['x' => 2], ['readConcern' => $rc]);
+$cursor = $manager->executeQuery(NS, $query);
+
+var_dump(iterator_to_array($cursor));
+
+$rc = new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::MAJORITY);
+$query = new MongoDB\Driver\Query(['x' => 2], ['readConcern' => $rc]);
+
+echo throws(function() use ($manager, $query) {
+    $manager->executeQuery(NS, $query);
+}, 'MongoDB\Driver\Exception\RuntimeException'), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+array(1) {
+  [0]=>
+  object(stdClass)#%d (%d) {
+    ["_id"]=>
+    int(2)
+    ["x"]=>
+    int(2)
+  }
+}
+OK: Got MongoDB\Driver\Exception\RuntimeException
+Majority read concern requested, but server was not started with --enableMajorityReadConcern.
```

Comment:
I think your environment might be not complete.

Can you merge https://github.com/mongodb/mongo-php-driver/pull/160 and run

    $ vagrant provision mo
    $ vagrant ssh mo
    $ sudo mongo-orchestration -f mongo-orchestration-config.json -b 192.168.112.10 --enable-majority-read-concern stop
    $ sudo mongo-orchestration -f mongo-orchestration-config.json -b 192.168.112.10 --enable-majority-read-concern start
    $ logout
    $ php scripts/start-servers.php
    $ php -n -c. tests/replicaset/readconcern-001.phpt


I get 
```
php -n -c. tests/replicaset/readconcern-001.phpt
--TEST--
ReadConcern: MongoDB\Driver\Manager::executeQuery() with readConcern option (find command)
--SKIPIF--
--FILE--
array(1) {
  [0]=>
  object(stdClass)#6 (2) {
    ["_id"]=>
    int(2)
    ["x"]=>
    int(2)
  }
}
FAILED: Expected MongoDB\Driver\Exception\RuntimeException thrown, but no exception thrown!

===DONE===
```

Which I consider to be correct, considering mongod is supposed to be running correctly.